cucumber/api/features/secrets_batch.feature

```diff
@@ -53,20 +53,28 @@ Feature: Batch retrieval of secrets
 
   Scenario: Fails with 404 if variable_ids param has some blank items
     When I GET "/secrets?variable_ids=cucumber:variable:secret1,,,cucumber:variable:secret2"
-    Then the HTTP response status code is 404
+    Then the HTTP response status code is 422
```

Comment:
looks like you are breaking the API. did @netacoral approve this? i'm ok with leaving it on 404.

Reply:
It is the same as only blank requests. @InbalZilberman @netacoral What do you say?

Reply:
looping in @izgeri as well

Reply:
@izgeri @orenbm I added verification that batch request doesn't contain nameless secrets ('/?variable_ids=var1,,,,,,var2'). Previously, such a request got a 404 response. In the test file `secrets_batch.feature` you can find that a request with '/?variable_ids=,,,,,,' returns `422`. The verification change caused this API change. Is that OK?

Reply:
@orenbm thank you pointing this out and reaching out.
If you look few tests above when providing ",,," as var ids we give 422 as our response code, hence this change is not breaking behaviour. The fact that this test provided 404 is actually a mystery from me. 
I think that returning 422 here is the right action and actually the previous test is wrong.
WDYT?